tests/unit/command/test_plots.py

```diff
@@ -297,3 +298,16 @@ def test_plots_diff_json(dvc, mocker, capsys):
     show_json_mock.assert_called_once_with(renderers, "out")
 
     render_mock.assert_not_called()
+
+
+def test_show_json_requires_out(mocker):
```

Comment:
This test may not be useful in the future and tests private function. Maybe we can just test `--json` through cli?